tests/Doctrine/Tests/Common/Annotations/CachedReaderTest.php

```diff
@@ -12,7 +12,7 @@ class CachedReaderTest extends AbstractReaderTest
 {
     private $cache;
 
-    public function testIgnoresStaleCache()
+    public function testIgnoresStaleCache() :void
```

Comment:
Please use a consistent style.  Some classes use ` : void`, this one uses ` :void`. IIRC, @Ocramius prefers `() : <type>`, so you should use that.